MAINTAINERS.md

```diff
@@ -1 +0,0 @@
-* Matthias Rampke <mr@soundcloud.com>
```

Comment:
This file is mandated by the [Prometheus governance](https://prometheus.io/governance/).